packages/stacks-classic/lib/components/pagination/pagination.a11y.test.ts

```diff
@@ -33,6 +33,6 @@ describe("pagination", () => {
             `,
         },
         tag: "nav",
-        additionalAssertions: [WCAGNonTextContrast],
+        // additionalAssertions: [WCAGNonTextContrast],
```

Comment:
Since we're not relying on background color to denote the selected state, we can remove this line and the import altogether.